django/db/models/aggregates.py

```diff
@@ -123,13 +126,14 @@ def as_sql(self, compiler, connection, **extra_context):
                 except FullResultSet:
                     pass
                 else:
-                    template = self.filter_template % extra_context.get(
-                        "template", self.template
-                    )
+                    extra_context = {
+                        **extra_context,
+                        "template": self.filter_template
+                        % extra_context.get("template", self.template),
+                    }
```

Comment:
Adding parentheses might make this this easier to grok while conforming to `black`

```suggestion
                    extra_context = {
                        **extra_context,
                        "template": (
                            self.filter_template % extra_context.get("template", self.template)
                         )
                    }
```